src/angular-local-storage.js

```diff
@@ -302,7 +302,10 @@ angularLocalStorage.provider('localStorageService', function() {
           expiryDate.setTime(expiryDate.getTime() + (-1 * 24 * 60 * 60 * 1000));
           expiry = "; expires=" + expiryDate.toGMTString();
           value = '';
-        } else if (isNumber(daysToExpiry) && daysToExpiry !== 0) {
+          //if the value is undefined set the cookie to session cookie
+        } else if (daysToExpiry == undefined) {
```

Comment:
This might not be a good solution. What if I want to set a cookie but I don't want to specify an `expiry` because I want it to use the value defined during the config phase?

I think that introducing this will be a breaking change because you are changing the expected behavior.